<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecialNumIU</class>
 <widget class="QWidget" name="SpecialNumIU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>特定时间段IU均值</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../qtResource/resource.qrc">
    <normaloff>:/icons/icons/预览.png</normaloff>:/icons/icons/预览.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="CardWidget" name="CardWidget_2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,2">
          <item row="0" column="0">
           <widget class="CaptionLabel" name="CaptionLabel">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>开始时间</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CalendarPicker" name="CalendarPicker"/>
          </item>
          <item row="0" column="2">
           <widget class="TimeEdit" name="TimeEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="CaptionLabel" name="CaptionLabel_2">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>截止时间</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="CalendarPicker" name="CalendarPicker_2"/>
          </item>
          <item row="1" column="2">
           <widget class="TimeEdit" name="TimeEdit_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PrimaryPushButton" name="PrimaryPushButton">
          <property name="text">
           <string>显示图像</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutRight">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsView_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarPicker</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../qtResource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
